configs/components/curl.rb

```diff
@@ -0,0 +1,45 @@
+component 'curl' do |pkg, settings, platform|
+  pkg.version '7.46.0'
+  pkg.md5sum '230e682d59bf8ab6eca36da1d39ebd75'
+  pkg.url "http://buildsources.delivery.puppetlabs.net/curl-#{pkg.get_version}.tar.gz"
```

Comment:
Is there a publicly available place to get this source?

Reply:
Thanks for the information. I had noticed similarly something like (https://github.com/puppetlabs/pl-build-tools-vanagon/blob/master/configs/components/sysroot.rb#L42) which is not obvious what that tarball even is or where one might could find it outside of the internal Puppetlabs repositories.

Reply:
@ehelms We generate that sysroot ourselves I believe.

/cc @ScottGarman

Reply:
@ehelms when it comes to sysroot generation, you should be able to find scripts in https://github.com/puppetlabs/pl-build-tools-vanagon/tree/master/scripts that can be used to try to re-create those tarballs (generally from a minimal install of the OS). For platforms such as RHEL and SLES I think there may be licensing issues to making those public. If you wanted access to the Debian/Ubuntu ones, let me know and I can share them with you if wanted.